src/nvim/os/time.c

```diff
@@ -46,26 +47,45 @@ void os_delay(uint64_t milliseconds, bool ignoreinput)
     }
     LOOP_PROCESS_EVENTS_UNTIL(&main_loop, NULL, (int)milliseconds, got_int);
   } else {
-    os_microdelay(milliseconds * 1000);
+    os_microdelay(milliseconds * 1000u, ignoreinput);
```

Comment:
Is this a problem? Currently I can see six instances where it might make any difference.

search.c
2148:        os_delay(p_mat * 100L, false);

syntax.c
7523:  os_delay(cnt == 99 ? 40L : (long)cnt * 50L, false);

if_cscope.c
1897:      os_delay(50L, false);       /* sleep 50 ms */
1928:          os_delay(50L, false);           /* sleep 50ms */

normal.c
628:  os_delay(3000L, false);             // wait up to three seconds

edit.c
1868:      os_delay(2000L, false);